### PR TITLE
Remove upper constraints

### PR DESCRIPTION
**Pull request by mit-mit**
## Description

Align plugins with new templates, removing unused upper constraint

## Related Issues

https://github.com/flutter/flutter/pull/68437

### CODE REVIEW

**Comment by mit-mit** [comment]
cc @amirh

File: packages/path_provider/path_provider/pubspec.yaml [hunk 1]
```diff
@@ -41,4 +41,4 @@ dev_dependencies:
 
 environment:
   sdk: ">=2.1.0 <3.0.0"
-  flutter: ">=1.12.13+hotfix.5 <2.0.0"
+  1.12.13+hotfix.5"
```
Comment (ditman) [inline packages/path_provider/path_provider/pubspec.yaml:44]: Ooops, typo here:

```suggestion
  flutter: ">=1.12.13+hotfix.5"
```

**Review by ditman — COMMENTED** [review]
This LGTM in general, wouldn't we need to bump the patch version of each plugin + their CHANGELOG so they become publishable in pub.dev?

**Review by amirh — COMMENTED** [review]
Can you add CHANGELOG entries and bump the versions?

**Comment by mit-mit** [comment]
@googlebot I fixed it

**Comment by mit-mit** [comment]
Abandoning for https://github.com/flutter/plugins/pull/3236

**Pull request closed without merging**